lib/resources/tags.js

```diff
@@ -0,0 +1,37 @@
+const taggableEntities = [
+  {
+    keyName: 'Ticket',
+    urlPrefix: 'tickets',
+  },
+  {
+    keyName: 'User',
+    urlPrefix: 'users',
+  },
+  {
+    keyName: 'Organization',
+    urlPrefix: 'organizations',
+  },
+];
+
+module.exports = ({ api }) => {
+  function initTagHelpersForEntities() {
+    const actionsPerEntity = taggableEntities.map(entity => ({
```

Comment:
Do you always use this technique to generate client functions?
It is effective but could be harder to read for another dev.